admin-dev/themes/new-theme/js/pages/order/create/cart-rules-renderer.ts

```diff
@@ -123,7 +123,7 @@ export default class CartRulesRenderer {
     const $template = $(
       $(createOrderMap.cartRulesNotFoundTemplate).html(),
     ).clone();
-    this.$searchResultBox.replaceWith($template);
+    this.$searchResultBox.append($template);
```

Comment:
Cloning the template above is useless since jQuery creates a new DOM Element from HTML content.
Wrapping HTML content in jQuery object is useless as well.
Here we can append HTML content directly: https://api.jquery.com/append/#append-content-content
```suggestion
    this.$searchResultBox.append(
        $(createOrderMap.cartRulesNotFoundTemplate).html()
    );
```

Reply:
Fixed